specs/slider/slider.md

```diff
@@ -0,0 +1,181 @@
+# Slider + Slider-Label
+
+## Overview
+
+The `slider` component allows the user to move one or two `thumb` elements along a vertical or horizontal axis to select a value or range. `slider` is also considered an input field and must work within a `<form>` as other input fields do.
+
+### Background
+
+Just styling `<input type="range"/>` and using `<datalist>` for the tickmarks was considered but the standards aren't supported enough to allow for the level customization we want to provide.
+
+** see [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range)
+
+### Use Cases
+
+- *A customer using the component on a web page.*
+On a web page the customer drags the slider from min to max values ultimately setting the value of the component to a value constrained by step and min and max values.
+
+- *A developer building an app with the component and interacting through HTML/CSS/JavaScript.*
+A developer can inject labels on the slider into the default slot, if they use <slider-label> elements those will be styled a specific way internally and any part of that style can be overridden. Styles and items intending to be placed in the default slot can be replaced as needed.
+
+- *A designer customizing the component.*
+A designer can override any internal styling applied to the slotted labels, the background track, the slider track and the thumb(s) if appropriate.
+  
+### Features
+- Implement your own lower and upper thumb UI, default provided
+- Implement your own background and slider track UI, default provided
+- Handle the `slider`'s `onValueChange` callback to be notified when the value has changed.
+- Customize the tick marks along the track
+- Include slider in form based operations
+
+### Prior Art/Examples
+- [FAST-DNA (React)](https://explore.fast.design/components/slider)
+- [Material UI](https://material-ui.com/components/slider/)

```

Comment:
Are these the only two examples that we can hunt down? Would be ideal to have a few more if we can find them.

Reply:
ok will look for a couple more

Reply:
added Atlassian and Lightning Design System. Which both do not incorporate upper and lower thumbs or vertical/horizontal orientation